src/PrestaShopBundle/Resources/config/services/adapter/data_configuration.yml

```diff
@@ -261,3 +261,9 @@ services:
       class: 'PrestaShop\PrestaShop\Core\CreditSlip\CreditSlipOptionsConfiguration'
       arguments:
         - '@prestashop.adapter.legacy.configuration'
+
+    prestashop.core.feature_flags.configuration:
```

Comment:
There's a little inconsistency between the service name and the class
I like the class name, don't get me wrong, we can't exactly claim it as a regular configuration service But the service name should reflect the class name as well

Reply:
Fixed in https://github.com/PrestaShop/PrestaShop/pull/23703/commits/bb2f40a2a10a847f5f1144549532bff524cc3074